src/plugins/jsx/index.js

```diff
@@ -37,6 +37,19 @@ tt.jsxTagEnd.updateContext = function(prevType) {
   }
 };
 
+function isFragment(
+  object:
+    | ?N.JSXOpeningElement
+    | ?N.JSXClosingElement
+    | ?N.JSXOpeningFragment
+    | ?N.JSXClosingFragment,
```

Comment:
Fragments cannot be self-closing, but it might be good to just accept any `N.JSXElement` to make it easier to test against arbitrary elements.

Although this is an internal use only function, so maybe it doesn't need to be extra ergonomic 🤔